### ISSUE TITLE
Balance not showing

### ISSUE THREAD

**Issue by Jpatten819** [stratisproject/StratisCore#468]
I sent Strat Tokens to a Stratis Core 2.2.1  wallet, the chain says its confirmed sent, but no balance is there in my wallet. I use a Mac book, and I sent my Strat from an Stratis-QT wallet. Want to move this  to the Strax wallet but cannot proceed. Any info would be greatly appreciated

**Comment by dev0tion** [comment]
Hi @Jpatten819, I think you want to swap to STRAX right?
In that case you will have to use your Qt wallet as the source for the swap.
The first swap window expired 12 Nov, transactions after that do not count anymore.

Please keep an eye out for an announcement on how to proceed with the second window of the swap.

**Comment by Jpatten819** [comment]
Yes I do want to swap, the issue is I sent the Stratis to the core wallet and its no longer in my QT wallet, The block chain says that the Strat sent. How would I use my QT wallet as the source?

**Comment by dev0tion** [comment]
We are using the UTXO set of the block closest to 12 Nov 9AM GMT as a reference point.
At that point in time, your STRAT was not send to Stratis Core yet, so you will have to use Qt to do the swap.

**Comment by Jpatten819** [comment]
Great, thank you for making that clear. Is there a link that I could get explaining how I would do use the QT for a swap? I haven't been able to find any info besides using the Strat core swap function.

**Comment by dev0tion** [comment]
No, that information has not been released yet. Keep an eye out for our announcements. There will be a blog post on our website with more information.

**Comment by Jpatten819** [comment]
Thank you very much!
